pg_duckdb/results/c6a.4xlarge.json

```diff
@@ -1,57 +1,57 @@
 {
     "system": "pg_duckdb",
-    "date": "2024-11-18",
+    "date": "2025-03-05",
     "machine": "c6a.4xlarge, 500gb gp2",
     "cluster_size": 1,
 
     "tags": ["column-oriented", "PostgreSQL compatible"],
 
     "load_time": 0,
-    "data_size": 14820690158,
+    "data_size": 0,
```

Comment:
I couldn't figure out the load time and data sizes. If someone knows, please send a PR.